### DISCUSSION TITLE
Tailwind CSS styling is not working on Astro site

### DISCUSSION THREAD

**Discussion by Albertyhu** [tailwindlabs/tailwindcss#13451] [Help]
I've built a site with Webflow and transferred the HTML and CSS files into Astro. 

I used Tailwind CSS to style certain components.

This is done with the combination of external style sheets. 

The Tailwind styling works in development mode, but not in production mode. 

To show you an example of what I am talking about, here is a page I built:
https://marks-painting.netlify.app/services/unique-custom-murals-and-artwork

Scroll down to the section where it looks like a series of pictures taking up the fullscreen. 
They are supposed to be placed in a grid utilizing the grid-template-columns property. 

Here is the code that is used to render the webpage: 
https://github.com/Albertyhu/painting-company/blob/main/src/pages/services/unique-custom-murals-and-artwork.astro

Here are the files used to render the gallery on the webpage:
https://github.com/Albertyhu/painting-company/tree/main/src/components/gallery-panel

I have followed the instructions in this document to install Tailwind into Astro:
https://tailwindcss.com/docs/guides/astro

I appreciate any help I can get.

**Comment by kunalx2901** [comment]
kindly use the tailwind CLI in the html sheet then you will be able to see tailwind after the development !!

**Reply by Albertyhu** [reply]
Thanks for the suggestion, @kunalsharma0101 
It works now.
